Orange/projection/setup.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import numpy
+
+
+def configuration(parent_package='', top_path=None):
+    from numpy.distutils.misc_util import Configuration
```

Comment:
Is there a reason this is imported inside the function instead of at the top? I see that `setup.py`s in other dirs do the same, but don't know if there is a reason behind it or are we just copy pasting and propagating this.

I ask because moving it up and changing `import numpy` to `from numpy import get_include` (and using that below) might be nice enough and make lint pass as well and avoid the ugly red cross on travis :)
(it is complaining that numpy is not used, which is a bit strange anyway)

Reply:
Your suspicion is correct, I copied this from other setups. :) If we're sure this import doesn't need to be local, we should fix all setups. Which implies it doesn't belong to this PR. :)